Dockerfile

```diff
@@ -12,4 +12,6 @@ RUN ln -s /portus/docker/crono-supervisord.conf /etc/supervisor/conf.d/crono.con
 
 ENV COMPOSE=1
 
+ADD . /portus
```

Comment:
That's correct for development, but I'm planning to use docker containers to deploy to production as well. So I need to have the code inside the container.
The only thing that differs is the volume mount and the environment variables (RAILS_ENV)